src/test/test.js

```diff
@@ -464,12 +464,17 @@ describe('Registrar', () => {
                     assert.equal(entryErr1, null);
                     assert.equal(entryResult1.name, 'foo');
                     assert.ok(entryResult1.deed.owner.slice(0,2) == '0x');
-                    assert.equal(entryResult1.mode, 'forbidden-can-invalidate');
+                    // the `forbidden-can-invalidate` mode does not exist in the registrar 
+                    // but is useful in this context where we know the plain text name
+                    assert.equal(entryResult1.mode, 'forbidden-can-invalidate'); 
                     debugger;
+                    // status `2` corresponds to 'owned'
                     assert.equal(entryResult1.status, 2);
                     registrar.invalidateName('foo', { from: accounts[1], gas: 4700000 }, (invalidateErr, invalidateResult) => {
```

Comment:
This call is failing, despite the checks in the previous steps to verify that is should be possible to invalidate.